src/views/Comments.js

```diff
@@ -0,0 +1,137 @@
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable no-shadow */
+import React, { Component } from "react";
+import Textarea from "react-textarea-autosize";
+import PropTypes from "prop-types";
+import moment from "moment";
+import authorImage from "../assets/img/user.jpg";
+import Button from "../components/common/Buttons/BasicButton";
+import DeleteDialogue from "../components/common/Dialog/DeleteDialogue";
+import heartIcon from "../assets/img/heart.svg";
+
+export class Comments extends Component {
+  state = {
+    editMode: false,
+    ind: null
```

Comment:
@niomwungeri-fabrice I am confused about the name of this variable. Is it `index` or `id`? I suggest that you make more descriptive of what it stands for.